VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx

```diff
@@ -469,13 +469,26 @@ function PrescriptionHistory({ navigation, route }: PrescriptionHistoryProps) {
     )
   }
 
-  const getRequestRefillButton = () => (
-    <FloatingButton
-      testID="refillRequestTestID"
-      label={t('prescription.history.startRefillRequest')}
-      onPress={() => navigateTo('RefillScreenModal', { refillRequestSummaryItems: undefined })}
-    />
-  )
+  const getRequestRefillButton = () => {
+    if (
+      prescriptionInDowntime ||
```

Comment:
Should these be handled in the initial loading state rather than only being conditional for showing the request button? If this were to load the way we expect, I think the screen will show after the loading spinner and then the button pops in some time after that

Reply:
Talked to Adryien async and we identified that the footer content displays while the loading spinner is active, so the current implementation outside of the screen loading logic makes sense.